tests/codex/testvalidation.nim

```diff
@@ -12,19 +14,32 @@ import ./helpers
 asyncchecksuite "validation":
   let period = 10
   let timeout = 5
-  let maxSlots = 100
+  let maxSlots = MaxSlots(100)
+  let validationGroups = ValidationGroups(8).some
   let slot = Slot.example
   let proof = Groth16Proof.example
   let collateral = slot.request.ask.collateral
 
   var validation: Validation
   var market: MockMarket
   var clock: MockClock
+  var groupIndex: uint16
+
+  proc initValidationConfig(maxSlots: MaxSlots,
+                            validationGroups: ?ValidationGroups,
+                            groupIndex: uint16 = 0): ValidationConfig =
+    without validationConfig =? ValidationConfig.init(
+      maxSlots, groups=validationGroups, groupIndex), error:
+      raiseAssert fmt"Creating ValidationConfig failed! Error msg: {error.msg}"
+    validationConfig
 
   setup:
+    groupIndex = groupIndexForSlotId(slot.id, !validationGroups)
     market = MockMarket.new()
     clock = MockClock.new()
-    validation = Validation.new(clock, market, maxSlots)
+    let validationConfig = initValidationConfig(
+        maxSlots, validationGroups, groupIndex)
+    validation = Validation.new(clock, market, validationConfig)
```

Comment:
Needed? From what I can see, all the tests create their own instances of `validationConfig` and `validation`.

Reply:
Ignore me, I see this is used in the existing tests.